package.json

```diff
@@ -83,7 +83,7 @@
     "enquirer": "^2.3.2",
     "eslint": "^6.6.0",
     "eslint-config-airbnb": "^18.0.1",
-    "eslint-config-prettier": "^6.5.0",
+    "eslint-config-prettier": "^6.6.0",
```

Comment:
Hard to tell what changed between releases: https://github.com/prettier/eslint-config-prettier/releases/tag/v6.6.0